features/svg-animations.yml

```diff
@@ -0,0 +1,99 @@
+name: SVG animations
+description: The `<animate>`, `<animateMotion>`, and `<animateTransform>` SVG elements declaratively animate SVG elements. Also known as SMIL.
+spec: https://svgwg.org/specs/animations/
+caniuse: svg-smil
+group:
+  - animation
+  - svg
+# Most features are quite old, but only became baseline when Edge adopted Chromium.
+status:
+  compute_from: api.SVGAnimateElement
```

Comment:
Can we pin to `svg.elements.{animate,animateMotion,animateTransform}` instead? That'll align more closely with caniuse and guarantee to not under-report the entrypoints mentioned in the description.